docs/reference/pip_install.rst

```diff
@@ -128,6 +128,46 @@ You can also refer to constraints files, like this::
 
     -c some_constraints.txt
 
+.. _`Example Requirements File`:
+
+Example Requirements File
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use ``pip install -r dev-requirements.txt`` to install::
```

Comment:
I'd prefer it be called "example-requirements.txt"  instead of "dev".  I don't want this to be construed as an actual recommendation for a "devopment" process.